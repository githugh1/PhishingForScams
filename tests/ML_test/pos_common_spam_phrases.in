{
  "email_subject": "Enlarge Your Savings",
  "email_body": "Guaranteed to enlarge your savings...",
  "sender": "offers@email.com"
}
