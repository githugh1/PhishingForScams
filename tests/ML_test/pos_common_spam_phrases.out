{
  "is_spam": true,
  "confidence": 0.85,
  "message": "This email is likely spam."
}
